src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaFactoryTests.cs

```diff
@@ -137,6 +139,13 @@ public static void ImportSubjectKeyPublicInfo_AlgorithmErrorsInAsn()
                 SubjectPublicKey = new byte[SlhDsaAlgorithm.SlhDsaSha2_128s.PublicKeySizeInBytes]
             };
 
+            SlhDsaTestHelpers.AssertImportSubjectKeyPublicInfo(
+                import => AssertThrowIfNotSupported(() => Assert.Throws<CryptographicException>(() => import(spki.Encode()))));
+
+            AsnWriter nullWriter = new AsnWriter(AsnEncodingRules.DER);
+            nullWriter.WriteNull();
+            spki.Algorithm.Parameters = nullWriter.Encode();
```

Comment:
It's `new byte[] { 0x05, 0x00 }`.  There's no need to wake up AsnWriter over and over and over again.

Reply:
Just to clarify, you mean cache `new byte[] { 0x05, 0x00 }` outside this method?

Reply:
I 80% expect we have already done so.  But might not have since we usually just embed full blobs, not have AsnWriter re-write them every time.

If there's nowhere that already exists that makes sense to put it, you can just declare the array inline.  It just feels exceedingly wasteful to wake up AsnWriter to allocate a 2 byte array.

Reply:
I did it once for ML-KEM. I suppose we could stick it in some generic place, but it doesn't seem worth too much heart ache if it's easier to just repeat it.

https://github.com/dotnet/runtime/blob/251d097ebc7400af594f9469078eef9663b78fb4/src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.cs#L17